t/data/crawler_stories/gv/9.json

```diff
@@ -1,548 +1,537 @@
ion rdf:about=\"http://globalvoicesonline.org/2009/06/08/hong-kong-say-no-to-cctvb/\"\n    dc:identifier=\"http://globalvoicesonline.org/2009/06/08/hong-kong-say-no-to-cctvb/\"\n    dc:title=\"Hong Kong: Say No to CCTVB\"\n    trackback:ping=\"http://globalvoicesonline.org/2009/06/08/hong-kong-say-no-to-cctvb/trackback/\" />\n</rdf:RDF>\t\t\t\t-->\n\t\t\t</div>\n\t\t\t\n\t\t\t\t\t\t\n\t\t\t\n<!-- You can start editing here. -->\n<div id=\"comments\">\n\n\t\t\n\t<h3>9 comments</h3> \n\n\t<!--comments nav - only shows at the top after you are off the first page-->\n\t<div class=\"navigation comments-nav\">\n\t\t\t</div>\n\t\n\t<ul class=\"commentlist\">\n\t\t\t\n\t\t<li class=\"comment even thread-even depth-1\" id=\"li-comment-1569791\">\n\t\t\t<div class=\"commentdiv\" id=\"comment-1569791\">\n\t\t\t\t<div class=\"comment-author vcard\">\n\t\t\t\t\t<img alt='' src='http://www.gravatar.com/avatar/9eb9698a9e15ac7fdf255f54f779aa7d?s=48&amp;d=http%3A%2F%2Fwww.gravatar.com%2Favatar%2Fad516503a11cd5ca435acc9bb6523536%3Fs%3D48&amp;r=G' class='author-profile-pic avatar avatar-48 photo' height='48' width='48' />\t\n\t\t\t\t\t<cite class=\"comment-author\">\n\t\t\t\t\t\t<a href='' rel='external nofollow' class='url'>mao</a>\t\t\t\t\t</cite>\n\t\t\t\t</div>\n\t\t\t\t\t\n\t\t\t\t<p>Hongkong is useless. It has no real hightech knowedge base to help the motherland stronger. Well, at least Taiwan has strength in semiconductor knowledge which China can use and gain</p>\n<p>All it has is garbage middleman financial stuffs and this type of BS, white man&#8217;s game of democratic movement.</p>\n\t\n\t\t\t\t<div class=\"comment-meta commentmetadata\">\n\t\t\t\t\t<a href=\"http://globalvoicesonline.org/2009/06/08/hong-kong-say-no-to-cctvb/comment-page-1/#comment-1569791\"> # </a>\n\t\t\t\t\t<span class=\"comment-date\">2009-06-8 at 17:27 pm</span>\t\t\t\t</div>\n\t\t\t\t<div class=\"reply\">\n\t\t\t\t\t<a rel='nofollow' class='comment-reply-link' href='/2009/06/08/hong-kong-say-no-to-cctvb/?replytocom=1569791#respond' onclick='return addComment.moveForm(\"comment-1569791\", \"1569791\", \"respond\", \"78992\")'>Reply to this</a>\t\t\t\t</div>\n\t\t\t</div>\n\t\n<ul class='children'>\n\t\n\t\t<li class=\"comment byuser comment-author-oiwan bypostauthor odd alt depth-2\" id=\"li-comment-1570040\">\n\t\t\t<div class=\"commentdiv\" id=\"comment-1570040\">\n\t\t\t\t<div class=\"comment-author vcard\">\n\t\t\t\t\t<img alt='' src='http://www.gravatar.com/avatar/6f109828da91ea546c9b68eb56fed073?s=48&amp;d=http%3A%2F%2Fwww.gravatar.com%2Favatar%2Fad516503a11cd5ca435acc9bb6523536%3Fs%3D48&amp;r=G' class='author-profile-pic avatar avatar-48 photo' height='48' width='48' />\t\n\t\t\t\t\t<cite class=\"comment-author\">\n\t\t\t\t\t\t<a href='/author/oiwan/' class='url'>Oiwan Lam</a>\t\t\t\t\t</cite>\n\t\t\t\t</div>\n\t\t\t\t\t\n\t\t\t\t<p>yeah, let&#8217;s celebrate corruption instead.</p>\n\t\n\t\t\t\t<div class=\"comment-meta commentmetadata\">\n\t\t\t\t\t<a href=\"http://globa
```

Comment:
New `html_strip()` seems to mess up periods (`:` becomes `.`, `...` becomes `..`), add a period after every newline, add some extra spaces around periods. Paragraphs are gone too, everything seems to be put into a single line. Not sure what impact will it have for sentence tokenization.